fcrepo-http-api/src/main/resources/views/common-node-actions.vsl

```diff
@@ -3,122 +3,123 @@
 #set ($isRoot = $helpers.isRootResource($rdf, $topic))
 
 #if ($isVersion == true)
-  #set ($original = $helpers.getOriginalResource($topic))
-  <div class="alert alert-warning">
-    <span class="glyphicon glyphicon-warning-sign"></span>
-    This is a <strong>historic version</strong> and cannot be modified.
-  </div>
-  #if ($helpers.isRdfResource($rdf, $original, "http://fedora.info/definitions/v4/repository#", "Binary"))
-    <a href="$content" class="btn btn-success btn-lg"><span class="glyphicon glyphicon-download"></span> Download Content</a><br/><br/>
-  #end
+    #set ($original = $helpers.getOriginalResource($topic))
+<div class="alert alert-warning d-flex align-items-center" role="alert">
+    <!-- Optional icon if you include Bootstrap Icons:
+    <i class="bi bi-exclamation-triangle-fill me-2"></i> -->
+    <div>
+        This is a <strong>historic version</strong> and cannot be modified.
+    </div>
+</div>
+    #if ($helpers.isRdfResource($rdf, $original, "http://fedora.info/definitions/v4/repository#", "Binary"))
+    <a href="$content" class="btn btn-success btn-lg">
+        <!-- <i class="bi bi-download me-1"></i> -->
```

Comment:
Probably want to retain the icon. Bootstrap has it own icons, and while there's not an exact match, this one might be fine
https://icons.getbootstrap.com/icons/file-earmark-arrow-down/